lib/error_store/manager.rb

```diff
@@ -82,6 +84,21 @@ def store_error
       issue
     end
 
+    #match a release
+    def check_release(slug_commit, website)
+      last_release = website.releases.last
+      unless slug_commit.nil?
+        current_release = Release.create_with(website_id: website.id).find_or_create_by(version: slug_commit)
+        unless last_release.nil? || last_release.version == current_release.version
+          last_release.grouped_issues.update_all(:status => 2, :release_id => current_release.id )
+        end
+      end
+      release = current_release unless slug_commit.nil?
```

Comment:
nu are nici un sens ce ai facut aici. Asta trebuie sa fie sus 
`release = Release.create_with(website_id: website.id).find_or_create_by(version: slug_commit)` 
in iful' ala de mai sus. Nu vad de ce verifici de 2 ori chestia asta.